Dockerfile

```diff
@@ -9,12 +9,24 @@ RUN \
     lsblk \
     dmidecode \
     util-linux \
-    lm-sensors \
-    speedtest-cli &&\
+    lm-sensors &&\
   apk --no-cache add \
     raspberrypi \
     || true
 
+ARG TARGETPLATFORM
```

Comment:
`TARGETPLATFORM` is not set for non-`dockerx` builds. e.g. in the dev image, which is run from docker-compose (`yarn run dev --build`).

Reply:
Yeah true, but the alternative is `uname` which cannot differentiate between arm/v6 and arm/v7.